scripts/output/projects/FSDP_collect.R

```diff
@@ -325,6 +323,17 @@ for (i in 1:length(outputdir)) {
       y <- mbind(y,a)
     } else missing <- c(missing,nc_file)
 
+    ## Gridded temperature data from ISIMIP archive for relevant SSP/RCP
+    rcp <- ifelse(thisScen == "BAU", "ssp245", "ssp119")
```

Comment:
The [documentation](https://rdrr.io/r/base/ifelse.html) for `ifelse` says
> Further note that if(test) yes else no is much more efficient and often much preferable to ifelse(test, yes, no) whenever test is a simple true/false result, i.e., when length(test) == 1. 

So I'd suggest
```suggestion
    rcp <- if (thisScen == "BAU") "ssp245" else "ssp119"
```